op-challenger/game/keccak/fetcher/fetcher.go

```diff
@@ -53,13 +48,13 @@ func (f *InputFetcher) FetchInputs(ctx context.Context, blockHash common.Hash, o
 			return nil, fmt.Errorf("failed getting tx for block %v: %w", blockNum, err)
 		}
 		for _, tx := range block.Transactions() {
-			inputData, err := f.extractRelevantLeavesFromTx(ctx, oracle, signer, tx, ident)
+			inputData, err := f.extractRelevantLeavesFromTx(ctx, oracle, tx, ident)
 			if err != nil {
 				return nil, err
 			}
 			if inputData != nil {
```

Comment:
Since `inputData` changed to be a slice, we should probably change this to:
```suggestion
			if len(inputData) > 0 {
```

a `nil` slice in go is just an empty slice so `len(nil)` returns 0.  But `[]InputData{} != nil` so there is a difference between `nil` which acts like an empty slice and an actual empty slice (it becomes quite visible in JSON serialisation using the builtin JSON library).